ytfzf

```diff
@@ -92,9 +92,12 @@ pid_file="$cache_dir/.pid"
 #list of shortcuts to use in fzf
 #handle_custom_shortcuts will be called if defined (handle_custom_shortcuts can call handle_shortcuts to handle normal shortcuts)
 shortcuts="${shortcuts-alt-l,alt-t,alt-o,alt-v,alt-d,alt-m,alt-s,alt-enter}"
+# these keys allow the use of shortcuts without leaving fzf
+persistent_shortcuts="${fzf_select_bindings_play-enter,alt-enter,alt-v,alt-d,alt-o}"
```

Comment:
maybe rename `fzf_select_bindings_play` to `fzf_bind_shortcuts`

`fzf_select_bindings_play` is a bit long and has a bit of an odd name.